package.json

```diff
@@ -10,7 +10,7 @@
     "docs-serve": "npm run docs-build && (cd website && npm run start)",
     "remove-locks": "lerna exec 'rm package-lock.json'",
     "install-locks": "lerna exec 'npm i --package-lock-only --no-audit' --parallel --no-sort",
-    "lint": "eslint example packages/api packages/checksum packages/cmrjs packages/cmr-client packages/common packages/deployment packages/ingest packages/s3-access-metrics tasks",
+    "lint": "eslint example packages/api packages/checksum packages/cmrjs packages/cmr-client packages/common packages/deployment packages/ingest packages/s3-replicator tasks",
```

Comment:
we didn't have s3-replicator in here before? Weird

Reply:
No, it was a new package Menno added.  I just noticed it wasn't being checked so I added it as part of this PR